### ISSUE TITLE
Could not find osmdroid-android.aar (org.osmdroid:osmdroid-android:5.4.1).

### ISSUE THREAD

**Issue by moonandroid** [MKergall/osmbonuspack#249]
Error:A problem occurred configuring project ':OsmAnd'.
> Could not find osmdroid-android.aar (org.osmdroid:osmdroid-android:5.4.1).
  Searched in the following locations:
      https://jcenter.bintray.com/org/osmdroid/osmdroid-android/5.4.1/osmdroid-android-5.4.1.aar

----------------------------------------------------------------------------------------------------------
// Top-level build file where you can add configuration options common to all sub-projects/modules.

buildscript {
    repositories {
        jcenter()
    }
    dependencies {
        classpath 'com.android.tools.build:gradle:2.1.2'
        classpath 'com.github.dcendents:android-maven-gradle-plugin:1.3'

        // NOTE: Do not place your application dependencies here; they belong
        // in the individual module build.gradle files
    }
}

allprojects {
    repositories {
        jcenter()
        maven { url "https://jitpack.io" }
    }
}

**Comment by moonandroid** [comment]
Please help me for this issue because existing user faces issue for version-5 updation.

**Comment by MKergall** [comment]
Please provide your build.gradle or check with the build.gradle from the Tutorial project. 
Looks like you omitted the "release" keyword : 
`compile 'org.osmdroid:osmdroid-android:5.4.1:release@aar'
`

**Comment by kaeuchoa** [comment]
I'm facing the same problem. 
My gradle file is just like @moonandroid 's and I've tried to add the _release_ to solve the problem but I had no success yet.
I also tried to use the lib installation listed as _1.2 Including as a local lib_ on the guide. The gradle build is successfully done, but I can't import or use any of its features in my project.

This is the error presented still:

Error:Could not find osmdroid-android.aar (org.osmdroid:osmdroid-android:5.4.1).
Searched in the following locations:
    https://jcenter.bintray.com/org/osmdroid/osmdroid-android/5.4.1/osmdroid-android-5.4.1.aar

Thanks for the attention.

**Comment by MKergall** [comment]
Have a close look on the Tutorial build.gradle. 
Try to download and compile the tutorial project itself. 
And if not solved, provide the build.gradle of your app, where you have compile directives (not the top-level build.gradle).

**Comment by kaeuchoa** [comment]
I got it working by the following: 
1. On my Build.gradle (top level), I added:
`dependencies{
   ...
   classpath 'com.github.dcendents:android-maven-gradle-plugin:1.3'
}` and

`repositories {
        jcenter()
        maven { url "https://jitpack.io" }
    }`
2. On my Build.gradle (Module:app), I added:
`compile 'com.github.MKergall:osmbonuspack:5.9'
 compile 'org.osmdroid:osmdroid-android:5.4.1:release@aar'`

It seems the conflit was that I was using another version of osmdroid. 
When I changed to ` compile 'org.osmdroid:osmdroid-android:5.4.1:release@aar'`, it worked normally.
